src/dbnode/encoding/m3tsz/encoder_test.go

```diff
@@ -211,13 +211,13 @@ func TestEncodeNoAnnotation(t *testing.T) {
 
 	startTime := time.Unix(1427162462, 0)
 	inputs := []ts.Datapoint{
-		{startTime, 12},
-		{startTime.Add(time.Second * 60), 12},
-		{startTime.Add(time.Second * 120), 24},
-		{startTime.Add(-time.Second * 76), 24},
-		{startTime.Add(-time.Second * 16), 24},
-		{startTime.Add(time.Second * 2092), 15},
-		{startTime.Add(time.Second * 4200), 12},
+		{Timestamp: startTime, Value: 12},
```

Comment:
haha finally gave in to the annoying warnings, eh :p

Reply:
Haha yeah hand was forced because of the new field in `Datapoint` struct